src/main/java/com/cburch/logisim/gui/chrono/ChronoPanel.java

```diff
@@ -59,6 +60,8 @@ public class ChronoPanel extends LogPanel implements Model.Listener {
   private JScrollPane leftScroll;
   private JScrollPane rightScroll;
   private JSplitPane splitPane;
+  private JButton SelButton;
```

Comment:
Variables should start with a lowercase letter.

Reply:
Normally, this is corrected